snmp/datadog_checks/snmp/data/default_profiles/ruckus-unleashed.yaml

```diff
@@ -0,0 +1,78 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+  - ruckus.yaml
+
```

Comment:
looks like we don't have the vendor metadata: https://github.com/DataDog/integrations-core/blob/53cffe451f3e373be676bd74df4a317f06cd4c26/snmp/datadog_checks/snmp/data/default_profiles/avtech.yaml#L4-L8